robolectric/pom.xml

```diff
@@ -46,6 +82,12 @@
     </dependency>
 
     <dependency>
+      <groupId>xmlpull</groupId>
```

Comment:
So, android has its own version of xmlpull, which is loaded by the instrumenting class loader which makes it incompatible with PowerMock (it uses xmlpull for serialization). Specifying xmlpull in pom explicitly resolves that.